nursery/encrypt-data-using-rc4-via-systemfunction032.yml

```diff
@@ -0,0 +1,22 @@
+rule:
+  meta:
+    name: encrypt data using RC4 via SystemFunction032
+    namespace: data-manipulation/encryption/rc4
+    authors:
+      - richard.weiss@mandiant.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    mbc:
+      - Defense Evasion::Obfuscated Files or Information::Encryption-Standard Algorithm [E1027.m05]
+      - Cryptography::Encrypt Data::RC4 [C0027.009]
+    references:
+      - https://doxygen.reactos.org/df/d13/sysfunc_8c_source.html
+      - https://blog.gentilkiwi.com/tag/systemfunction032
```

Comment:
can we find any public examples of this technique that we can reference as an `example`?

Reply:
Yes, I will add. Thank you for the hint. Will have a look this evening for at least one sample.

Reply:
added example and also created a PR for the testfile

Reply:
testfile uploaded and reference changed.